assembly/src/parsers/field_ops.rs

```diff
@@ -30,6 +30,19 @@ pub(super) fn parse_assert_eq(
     Ok(())
 }
 
+/// Appends EQZ ASSERT operation sequence to the span block.
+pub(super) fn parse_assert_eqz(
```

Comment:
minor nit: I'd probably name the function `parse_assertz` to keep it consistent with the instruction name.